### ISSUE TITLE
Not working on latest gta version

### ISSUE THREAD

**Issue by ghost** [LfxB/CustomRadioStations-GTAV#1]
![image](https://user-images.githubusercontent.com/22838328/101258284-cc8a5b80-3721-11eb-9cc8-1381b6bec374.png)

**Comment by ghost** [comment]
Native stations log:
`2/2/2019 1:08:09 AM : Game version: VER_1_0_1493_0_NOSTEAM
2/2/2019 1:08:09 AM : Checking all native and add-on radios...
2/2/2019 1:08:09 AM : Name: RADIO_01_CLASS_ROCK || Proper name: Los Santos Rock Radio
2/2/2019 1:08:09 AM : Name: RADIO_02_POP || Proper name: Non-Stop-Pop FM
2/2/2019 1:08:09 AM : Name: RADIO_03_HIPHOP_NEW || Proper name: Radio Los Santos
2/2/2019 1:08:09 AM : Name: RADIO_04_PUNK || Proper name: Channel X
2/2/2019 1:08:09 AM : Name: RADIO_05_TALK_01 || Proper name: West Coast Talk Radio
2/2/2019 1:08:09 AM : Name: RADIO_06_COUNTRY || Proper name: Rebel Radio
2/2/2019 1:08:09 AM : Name: RADIO_07_DANCE_01 || Proper name: Soulwax FM
2/2/2019 1:08:09 AM : Name: RADIO_08_MEXICAN || Proper name: East Los FM
2/2/2019 1:08:09 AM : Name: RADIO_09_HIPHOP_OLD || Proper name: West Coast Classics
2/2/2019 1:08:09 AM : Name: RADIO_24_VCFM || Proper name: Vice City FM
2/2/2019 1:08:09 AM : Name: RADIO_12_REGGAE || Proper name: Blue Ark
2/2/2019 1:08:09 AM : Name: RADIO_13_JAZZ || Proper name: Worldwide FM
2/2/2019 1:08:09 AM : Name: RADIO_14_DANCE_02 || Proper name: FlyLo FM
2/2/2019 1:08:09 AM : Name: RADIO_15_MOTOWN || Proper name: The Lowdown 91.1
2/2/2019 1:08:09 AM : Name: RADIO_20_THELAB || Proper name: The Lab
2/2/2019 1:08:09 AM : Name: RADIO_16_SILVERLAKE || Proper name: Radio Mirror Park
2/2/2019 1:08:09 AM : Name: RADIO_17_FUNK || Proper name: Space 103.2
2/2/2019 1:08:09 AM : Name: RADIO_18_90S_ROCK || Proper name: Vinewood Boulevard Radio
2/2/2019 1:08:09 AM : Name: RADIO_21_DLC_XM17 || Proper name: Blonded Los Santos 97.8 FM
2/2/2019 1:08:09 AM : Name: RADIO_22_DLC_BATTLE_MIX1_RADIO || Proper name: Los Santos Underground Radio
2/2/2019 1:08:09 AM : Please use the 'Name' name for your wheel organization lists (NativeWheels.cfg)! 'Proper name' is only for display purposes.
`

**Comment by ghost** [comment]
Worth noting that it is on the Epic games version

**Comment by Michael21107** [comment]
yeah, @LfxB I think this is outdated, considering you instruct people to install ScriptHookVDotNet **2**.10.12.0
